test/common.js

```diff
@@ -77,4 +70,19 @@ module.exports = {
         return response.body;
     },
 
+    async deleteAllViaApi(entities, entityType, { addDefaultVhost = true } = {}) {
+        entities = entities.filter(x => x && x.name && x.name.indexOf('amq.') === -1);
+        console.log(`=============================== CLEANUP: deleting ${entities.length} ${entityType} ===============================`);
+
+        await Promise.each(entities, async entity => {
+            await this.deleteViaApi(entityType, entity.name, { addDefaultVhost });
+        });
+    },
+
+    async deleteViaApi(entityType, name, { addDefaultVhost = true } = {}) {
+        let response = await superagent
+            .delete(`${API_URL}/${entityType}/${addDefaultVhost ? '%2F/' : ''}${name}`)
```

Comment:
hey I understand your suggestions but since it's not production code, I'd rather let it develop as need arises, rather than investing time in something that may never have real value.